### PR TITLE
[psud] Refactor unit tests; increase unit test coverage

### PR DESCRIPTION
**Pull request by jleveque**
- Encapsulate all PsuChassisInfo test cases in a TestPsuChassisInfo class
- Move TestPsuChassisInfo to its own file, test_PsuChassisInfo.py
- Increase coverage of PsuChassisInfo class
- Increase coverage of TestDaemonPsud class
- In all test scripts, use `import psud` rather than `from psud import *` and reference everything using the `psud.` namespace. Also do the same with `mock`
- Remove unnecessary try/except around import statements in psud
- Add unit tests for signal_handler
- Fix `TypeError: cannot concatenate 'str' and 'int' objects` in signal_handler

Overall psud unit test coverage increases from 51% to 78%.

Previous unit test coverage:

```
----------- coverage: platform linux, python 3.7.3-final-0 -----------
Name           Stmts   Miss  Cover
----------------------------------
scripts/psud     381    185    51%
Coverage HTML written to dir htmlcov
Coverage XML written to file coverage.xml
```

Unit test coverage with this patch:

```
----------- coverage: platform linux, python 3.7.3-final-0 -----------
Name           Stmts   Miss  Cover
----------------------------------
scripts/psud     360     81    78%
Coverage HTML written to dir htmlcov
Coverage XML written to file coverage.xml
```

### CODE REVIEW

**Review by lguohan — APPROVED** [review]

**Comment by jleveque** [comment]
@mprabhu-nokia: I have done some refactoring in this PR which impacts modular chassis. Can you please test this PR and ensure there are no regressions?

**Comment by jleveque** [comment]
@mprabhu-nokia: Can you please test and review these changes?

File: sonic-psud/scripts/psud [hunk 1]
```diff
@@ -606,25 +596,14 @@ class DaemonPsud(daemon_base.DaemonBase):
...
-
-        psu_chassis_info = self.psu_chassis_info
-        if psu_chassis_info.update_master_status():
-            log_on_status_changed(self, psu_chassis_info.master_status_good,
-                                  'PSU supplied power warning cleared: supplied power is back to normal.',
-                                  'PSU supplied power warning: {}W supplied-power less than {}W consumed-power'.format(
-                                      psu_chassis_info.total_supplied_power, psu_chassis_info.total_consumed_power)
-                                  )
-            psu_chassis_info._set_psu_master_led(psu_chassis_info.master_status_good)
+        self.psu_chassis_info.update_psu_chassis_info()
```
Comment (mprabhu-nokia) [inline sonic-psud/scripts/psud:599]: This should be self.psu_chassis_info.update_master_status()

Comment (jleveque) [reply sonic-psud/scripts/psud]: Thanks for the catch! Fixed in latest commit. Please test and review again.

Comment (jleveque) [reply sonic-psud/scripts/psud]: @mprabhu-nokia: Can you test and review again, please?

Comment (mprabhu-nokia) [reply sonic-psud/scripts/psud]: Yes, looks good.

**Review by mprabhu-nokia — APPROVED** [review]

File: sonic-psud/scripts/psud (hunk 1 above)
Comment (jleveque) [reply sonic-psud/scripts/psud]: Thank you!

**Review by lguohan — APPROVED** [review]

**Review by sujinmkang — APPROVED** [review]

**Pull request merged**